### ISSUE TITLE
high val_loss when tain using darknet53_weights.h5

### ISSUE THREAD

**Issue by 18814181500** [qqwweee/keras-yolo3#127]
@qqwweee Hi,I tained my dataset using darknet53_weights.h5 which got by using  "python convert.py -w darknet53.cfg darknet53.weights model_data/darknet53_weights.h5".
and I used model_data/darknet53_weights.h5 in train.py, but when tainning I got a high val_loss always,could you have any suggestions? Thank you.
Epoch 130/200
133/133 [==============================] - 5s 37ms/step - loss: 35.1921 - val_loss: 4100.4419
Epoch 131/200
133/133 [==============================] - 5s 37ms/step - loss: 35.0289 - val_loss: 4097.4834
Epoch 132/200
133/133 [==============================] - 5s 37ms/step - loss: 34.9217 - val_loss: 4094.5686
Epoch 133/200
133/133 [==============================] - 5s 37ms/step - loss: 34.5357 - val_loss: 4091.6243
Epoch 134/200
133/133 [==============================] - 5s 37ms/step - loss: 34.4389 - val_loss: 4088.6328
Epoch 135/200
133/133 [==============================] - 5s 37ms/step - loss: 34.1066 - val_loss: 4085.6099
Epoch 136/200
133/133 [==============================] - 5s 37ms/step - loss: 34.0354 - val_loss: 4082.8032
Epoch 137/200
133/133 [==============================] - 5s 37ms/step - loss: 33.5709 - val_loss: 4079.8918
Epoch 138/200
133/133 [==============================] - 5s 37ms/step - loss: 33.5741 - val_loss: 4077.2295

**Comment by EnJiang** [comment]
For those who have this problem:
Now I understood there are two stages of training: Train with frozen layers first, and then fine-tune. High val_loss in the first stage is normal, and there is no early stopping set for this stage. I change the epochs of this stage to 15 to speed up.

**Comment by zmlim** [comment]
@EnJiang  Hi, could you elaborate on how you did the training?
